components/chef-ui-library/src/molecules/chef-status-filter-group/chef-status-filter-group.scss

```diff
@@ -10,10 +11,10 @@ chef-status-filter-group {
     border-radius: 4px;
     font-size: 14px;
     display: inline-flex;
-    margin-bottom: 16px;
+    margin-bottom: 10px;
 
     &:not(:last-child) {
-      margin-right: 16px;
+      margin-right: 10px;
```

Comment:
@apriofrost i think our standard for space between buttons in automate has been at least 16px of space to prevent folks from interacting with buttons that they don't intend. could you tell me a bit more about the problem you're solving here?

Reply:
For reference, the "lean" version of these filter buttons is/has been a lot more compact: https://github.com/chef/automate/blob/783f03c60988c92a8b88e4dd1664b5fb85e03bb4/components/chef-ui-library/src/molecules/chef-status-filter-group/chef-status-filter-group.scss#L134-L153

Reply:
@susanev , I prefer it to be 10 px so that it's consistent with the space measurement we have for other UI molecules in this view. For example, the rows and headers in the tables has a shared distance of 10px. Same for the cards in the side nav. 
The idea is that by reducing the distance, the individual atom within a molecule feels like a group that shares the same look, function, and interactivity. While in between molecules, I set the space as 16px so they feel further apart from each other. 
In this example, all the different status filters form a group of filters, that serve the same purpose of showing an aggregated count of service groups in that status. I would worry more about the space between buttons in a group since the button size is usually much smaller than the status filter here. But since the status filter is already large enough, I won't worry about that. 
What do you think?

Reply:
I haven't pulled down the pr so maybe I misunderstood I was mostly worried about the accessibility issues caused by not having enough space between buttons, if that's not relevant here feel free to ignore my comment.

Reply:
No worries, feel free to take a look at it later when you have time and see how it feels to you :)